src/typeDefs/User.js

```diff
@@ -66,4 +75,16 @@ export default gql`
   type FavouriteBranch {
     favouriteBranchID: String!
   }
+
+  enum UserType {
+    EMPLOYEE
+    CUSTOMER
+    BRANDOWNER
+  }
+
+  enum EmployeeType {
+    MANAGER
+    STAFF
+    NONE
```

Comment:
NONE meaning what?
if brand owner and customer show not have this field right?

Reply:
I cannot put null for enum. so use none to replace

Reply:
ok